module/VuFind/src/VuFind/Controller/CombinedController.php

```diff
@@ -339,9 +342,6 @@ protected function adjustQueryForSettings($settings, $searchType = null)
         // Override the search type:
         $query->type = $searchType;
 
-        // Always leave noresults active (useful for 0-hit searches).
-        // Display or hide top based on include_recommendations setting.
```

Comment:
I think you might want to keep the comment about include_recommendations -- that's still accurate here, isn't it?

Reply:
Thanks, accidental delete.  Fixed.